.github/workflows/version-increment.yml

```diff
@@ -0,0 +1,31 @@
+name: version-increment
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'preview-[0-9]+\.[0-9]+\.[0-9]+'
+    paths-ignore:
+      - '.gitignore'
+      - '*.md'
+  push:
+    branches:
+      - 'gh-actions*'
+
+jobs:
+  helm:
+    runs-on: ubuntu-latest
+    name: 'version-increment-check'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: azure/setup-helm@v3
```

Comment:
Do you know if this is needed, or if chart-testing-action pulls in what's needed?

Reply:
Not needed, removed.